### PR TITLE
This is the Initial File for abhijith bio

### PR DESCRIPTION
**Pull request by abhijithrr**
Adding not required information

adding now a file to the complete address #10

### CODE REVIEW

File: githubabhijith-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Something about me
+ - I live in Bangalore
+ - I am from India
```
Comment (abhijithrr) [inline githubabhijith-bio.md:3]: From Which state are you?

File: githubabhijith-bio.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+ - I am new to Git
```
Comment (ernie-m) [inline githubabhijith-bio.md:4]: Do you not think that this instructor is going way too fast?

Comment (pandeysk) [inline githubabhijith-bio.md:4]: Are you serious? Where in Bangalore?

Comment (abhijithrr) [inline githubabhijith-bio.md:4]: @ernie-m , Yes I feel so. :+1:

**Pull request merged**